<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPrincipal</class>
 <widget class="QMainWindow" name="VPrincipal">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CONCESIONARIOS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Source/KWE.ico</normaloff>Source/KWE.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#VPrincipal{
background-color: rgb(255, 255, 255);
}
#imgPrincipalMenu{
border-radius:8px;
}
#btnKWRB{
background-color: rgb(232, 232, 232);
	color: rgb(0, 0, 0);
	border-radius:8px;
	font-size: 15px;
	font-weight: bold;
}
#btnKWRB::Hover{
background-color: rgb(209, 209, 209);
	color: rgb(0, 0, 0);
}
#btnKWESTE{
background-color: rgb(232, 232, 232);
	color: rgb(0, 0, 0);
	color: rgb(0, 0, 0);
	border-radius:8px;
	font-size: 15px;
	font-weight: bold;
}
#btnKWESTE::Hover{
background-color: rgb(209, 209, 209);
	color: rgb(0, 0, 0);
}
#btnKWKREI{
background-color: rgb(232, 232, 232);
	border-radius:8px;
	font-size: 15px;
	font-weight: bold;
}
#btnKWKREI::Hover{
background-color: rgb(209, 209, 209);
	color: rgb(0, 0, 0);
}
#btnKWS{
background-color: rgb(232, 232, 232);
	border-radius:8px;
	font-size: 15px;
	font-weight: bold;
}
#btnKWS::Hover{
background-color: rgb(209, 209, 209);
	color: rgb(0, 0, 0);
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>45</width>
    <height>45</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnKWRB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>KENWORTH RIO BRAVO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKWESTE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>KENWORTH DEL ESTE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKWKREI">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>KENWORTH KREI</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgPrincipalMenu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKWS">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>KENWORTH SONORA</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnKWRB</sender>
   <signal>clicked()</signal>
   <receiver>VPrincipal</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
